.gitmodules

```diff
@@ -58,3 +58,6 @@
 [submodule "src/c-ares"]
 	path = src/c-ares
 	url = https://github.com/ceph/c-ares.git
+[submodule "src/fmt"]
+	path = src/fmt
+	url = https://github.com/ceph/fmt.git
```

Comment:
@yuvalif why do you need another copy of libfmt? can we instead use src/seastar/fmt ?

Reply:
@tchaikov you are right , no need for another submodule